### PR TITLE
update(docs): add more comparison columns

### PR DESCRIPTION
**Pull request by okainov**
Sorry for the diff, it's barely readable.... Here is the summary of the changes:

- Add a few more columns: "Works with GitHub", "On-prem possible?" and "Project alive?". Motivation - some of the tools just don't work with Github (like Marge-bot for Gitlab or CommitQueue for SVN(?)), some of the repos are not maintained, archived or just not available.
- Update link for bors-ng to point to Github repo (for consistency with other items)
- Reorder the list a bit to move down not alive projects

### CODE REVIEW

File: docs/docs/prior-art-and-alternatives.md [hunk 1]
```diff
@@ -7,25 +7,25 @@ sidebar_label: Prior Art / Alternatives
...
-| <!-- 2021-08-28 --> <a rel="nofollow" href="https://mergequeue.com">MergeQueue</a>                                                                 | ✅           | ✅                   | ✅                          | ✅                           | ❌               | ❌                   | ❌          | ✅          | ❌                                        | 🤷‍        |
-| <!-- Unknown --> <a rel="nofollow" href="https://reporanger.com">Ranger</a> ‍                                                                      | ✅           | ❌ ‍                 | ❌ ‍                        | ✅                           | ❌               | ❌                   | ❌          | ✅          | ❌                                        | 🤷‍        |
+| Name                                                                                                                                               | Auto Merging | Auto Update Branches | Update Branches Efficiently | Works With Branch Protection | Works with Forks | Simple Configuration | Open Source | Hosted SaaS | Works with GitHub | On-prem possible? | Project alive? | Free for public and personal repositories | Language   |
```
Comment (sbdchd) [inline docs/docs/prior-art-and-alternatives.md:10]: looks good, thanks for adding the extra columns

one minor thing is I'd prefer the order to be chronological (with the exception of Kodiak) based on their creation data which is in the html comment in the first column

Comment (okainov) [reply docs/docs/prior-art-and-alternatives.md:10]: Done, reordered from new to old. Also updated the date for MergeQueue as it seemed to be in the future (end of August'21), the first blog article was around January, so I put January'21

**Comment by okainov** [comment]
CI complains about some formatting, but doesn't actually print the issues...

>Checking formatting...
>docs/prior-art-and-alternatives.md
>Code style issues found in the above file(s). Forgot to run Prettier?

Is there anything I need to do to fix it?

**Comment by sbdchd** [comment]
@okainov ah yeah that's the formatter, I can run it an merge it, thanks for the PR!

**Review by sbdchd — APPROVED** [review]

**Pull request merged**